exercises/isbn-verifier/description.md

```diff
@@ -9,27 +12,35 @@ The program should allow for ISBN-10 without the separating dashes to be verifie
 
 ## ISBN
 
-Let's take a random ISBN-10 number, say `3-598-21508-8` for this.
+The ISBN-10 format is 9 digits (0 to 9) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+
+```
+(x1 * 10 + x2 * 9 + x3 * 8 + x4 * 7 + x5 * 6 + x6 * 5 + x7 * 4 + x8 * 3 + x9 * 2 + x10 * 1) mod 11 == 0
+```
+
+If the result is 0, then it is a valid ISBN-10, otherwise it is invalid.
+
+## Example
+
+Let's take a random ISBN-10 number, for instance `3-598-21508-8`.
```

Comment:
It's not really a random ISBN-10 number.
This line is unnecessarily verbose.

Reply:
I agree, that's why I slimmed it down. Do you think it should be reduced even more?

Reply:
Yes.
"For the ISBN-10 3-598..."

Be careful using "ISBN Number" since it has the "ATM Machine" issue.

Reply:
RAS Syndrome! Good point - I'll strip that too.